src/Compilers/CSharp/Portable/Symbols/MissingCorLibrarySymbol.cs

```diff
@@ -79,5 +79,20 @@ internal override bool RuntimeSupportsNumericIntPtr
                 throw ExceptionUtilities.Unreachable;
             }
         }
+
+        internal override bool RuntimeSupportsByRefFields
+        {
+            get
+            {
+                // For now we assume that it is not supported by default
+                Debug.Assert((object)CorLibrary == this);
+                return false;
+            }
+            set
+            {
+                Debug.Assert((object)CorLibrary == this);
```

Comment:
Is this assert meaningful?

Reply:
Also still following the pattern established by Aleksey back in DIM: [https://github.com/dotnet/roslyn/pull/17927/files#diff-25f1d6b9209e1dd8d317520686107ee7a0eff64b4e4696570d8ff7eaa32079a9](https://github.com/dotnet/roslyn/pull/17927/files#diff-25f1d6b9209e1dd8d317520686107ee7a0eff64b4e4696570d8ff7eaa32079a9)

I would rather avoid variation unless there's a strong reason, as we're going to have to do this kind of test hook for many features moving forward.